sentry/src/routes/channel.rs

```diff
@@ -846,4 +973,156 @@ mod test {
+        insert_campaign(&app.pool, &DUMMY_CAMPAIGN)
+            .await
+            .expect("should insert the campaign");
+
+        // Setting the initial remaining to 0
+        let campaign_remaining = CampaignRemaining::new(app.redis.clone());
+        campaign_remaining
+            .set_initial(DUMMY_CAMPAIGN.id, UnifiedNum::from_u64(0))
+            .await
+            .expect("Should set value in redis");
+
+        let auth = Auth {
+            era: 0,
+            uid: ValidatorId::from(DUMMY_CAMPAIGN.creator),
+            chain: channel_context.chain.clone(),
+        };
+
+        let build_request = |channel_context: &ChainOf<Channel>, payouts: UnifiedMap| {
+            let payouts = ChannelPayRequest { payouts };
+
+            let body = Body::from(serde_json::to_string(&payouts).expect("Should serialize"));
+
+            Request::builder()
+                .extension(channel_context.clone())
+                .extension(auth.clone())
+                .body(body)
+                .expect("Should build Request")
+        };
+
+        // Add accounting for spender -> 500
+        update_accounting(
+            app.pool.clone(),
+            channel_context.context.id(),
+            auth.uid.to_address(),
+            Side::Spender,
+            UnifiedNum::from_u64(500),
+        )
+        .await
+        .expect("should update");
+
+        // Add spendable for the channel_context where total deposit -> 1000
+        let spendable = Spendable {
+            spender: auth.uid.to_address(),
+            channel: channel_context.context,
+            deposit: Deposit {
+                total: UnifiedNum::from_u64(1000),
+                still_on_create2: UnifiedNum::from_u64(0),
+            },
+        };
+
+        // Add accounting for earner -> 100
+        update_accounting(
+            app.pool.clone(),
+            channel_context.context.id(),
+            auth.uid.to_address(),
+            Side::Earner,
+            UnifiedNum::from_u64(100),
+        )
+        .await
+        .expect("should update");
+        // available for return is now 600
+
+        // Updating spendable so that we have a value for total_deposited
+        update_spendable(app.pool.clone(), &spendable)
+            .await
+            .expect("Should update spendable");
+
+        // Test with no body
+        let res =
+            channel_payout(build_request(&channel_context, UnifiedMap::default()), &app).await;
+        assert!(
+            matches!(res, Err(ResponseError::FailedValidation(..))),
+            "Failed validation because request body is empty"
+        );
+        // make a normal request and get accounting to see if its as expected
+
+        let mut payouts = UnifiedMap::default();
+        payouts.insert(*PUBLISHER, UnifiedNum::from_u64(500));
+
+        let res = channel_payout(build_request(&channel_context, payouts.clone()), &app).await;
+        assert!(
+            res.is_ok(),
```

Comment:
It will be more beneficial to assert_eq with Ok(SuccessResponse {}) instead of just `is_ok` this way we will also see an error if for some reason it fails or returns `success: false`.